File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TestMode.java
package org.firstinspires.ftc.teamcode;


import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
import com.qualcomm.robotcore.hardware.ColorSensor;
import com.qualcomm.robotcore.hardware.DcMotor;
import com.qualcomm.robotcore.hardware.OpticalDistanceSensor;
import com.qualcomm.robotcore.hardware.Servo;

/**
 * Created by nitro on 10/9/2017.
 */

@TeleOp(name = "Test")
public class TestMode extends LinearOpMode {
    @Override
    public void runOpMode() {
        ColorSensor jewelColorSensor = hardwareMap.colorSensor.get("jewelColorSensor");
        waitForStart();

        while (opModeIsActive()) {
            telemetry.addData("Blue", jewelColorSensor.blue());
            telemetry.addData("Red", jewelColorSensor.red());
            telemetry.update();
            idle();
        }
    }
}

File: TeamCode/src/main/java/com/github/actonroboticsteam/ftcapp/TestMode.kt
package com.github.actonroboticsteam.ftcapp

import com.qualcomm.robotcore.eventloop.opmode.Autonomous
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode
import com.qualcomm.robotcore.hardware.DcMotor

@Autonomous(name = "Test program")
class TestMode : LinearOpMode() {
    override fun runOpMode() {
        val robot = RobotConfig(hardwareMap)

        // wait for the start button to be pressed.
        waitForStart()

        while (opModeIsActive()) {
            robot.leftDriveMotor.power = 1.0
            robot.rightDriveMotor.power = 1.0
            idle()
        }
    }
}